helm/app/templates/cronJobs/restart-ccbc.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: restart-ccbc
+spec:
+  schedule: "0 2 * * *" # Run at 2 AM every day
```

Comment:
I was under the impression the cron timezones were set to UTC - looking at the other cronjob we've got setup and checking the time it says it was last run I think we'll want to change this. 2 AM PST is 10 AM UTC according to google

Reply:
Yes, totally slipped, it will run according to UTC. There seems to be a new timeZone parameter added in 4.12+, let's attempt and use that and check if it runs at 2am PST. If not then we can just adjust like the other ones.

https://docs.openshift.com/container-platform/4.12/rest_api/workloads_apis/cronjob-batch-v1.html